docker-compose.yml

```diff
@@ -48,8 +48,24 @@ services:
       - neo4j-logs:/logs
       - neo4j-import:/var/lib/neo4j/import
       - neo4j-plugins:/plugins
+  postgres:
+    container_name: mcs-postgres
+    environment:
+      POSTGRES_USER: "mcs"
+      POSTGRES_PASSWORD: "7EAdu7jJvZNxxrNZ"
+      POSTGRES_DB: "kg"
+    image: library/postgres:12.4
+    networks:
+      - mcs
+    ports:
+      - 5432:5432
```

Comment:
Only open to localhost (127.0.0.1:5432:5432).

Reply:
Gotcha changed